mirobo/ceil.py

```diff
@@ -104,4 +109,22 @@ def status(self) -> CeilStatus:
             "get_prop",
             properties
         )
+        properties_count = len(properties)
+        values_count = len(values)
+        if properties_count != values_count:
+            if properties_count == values_count+2:
+                _LOGGER.info(
+                    "The values of two properties are missing. Assumption: A "
+                    "Xiaomi Philips Light LED Ball is used which doesn't "
+                    "provide the property bl and ac.",
+                    properties_count, values_count)
+
+                values.extend((None, None))
```

Comment:
I admit inheritance of two devices (led ball and ceiling lamp) would be the better approach.

Reply:
This feels way too complicated and unpythonic. How about using `defaultdict` just below
```
defaultdict(lambda: None, zip(properties, values))
```
and having a warning() or debug() here if the amount of returned values is different from what was asked?